tasks/create_users.yml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Are you really going to create user role on elasticsearch using logstash? I suppose it's better to delegate role preparation to elasticsearch ansible role

Reply:
Changed behaivour to "Additional functionality" of the role